tests/unit/Claim/ClaimsTest.php

```diff
@@ -565,12 +568,6 @@ public function provideGetByRank() {
 				Statement::RANK_PREFERRED,
 				new Claims( array( $s2 ) ),
 			),
-			// s2 has RANK_PREFERRED, so doesn't match RANK_TRUTH
-			array(
-				new Claims( array( $s2 ) ),
-				Claim::RANK_TRUTH,
```

Comment:
Quite some changes in this patch set refer to that "truth" thingy, I honestly never understood. Can you please explain or, even better, split this into it's own pull request?